ndcube/mixins/sequence_plotting.py

```diff
@@ -966,8 +967,8 @@ def __init__(self, seq, plot_axis_index=None, axis_ranges=None, unit_x_axis=None
                     unit_x_axis = np.asarray(seq[0].wcs.wcs.cunit)[wcs_plot_axis_index]
                 # Get x-axis values from each cube and combine into a single
                 # array for axis_ranges kwargs.
-                x_axis_coords = _get_non_common_axis_x_axis_coords(seq.data, cube_plot_axis_index,
-                                                                   unit_x_axis)
+                x_axis_coords = _get_extra_coord_edges(_get_non_common_axis_x_axis_coords(seq.data, cube_plot_axis_index,
```

Comment:
Where is ```_get_non_common_axis_x_axis_coords``` coming from?

Reply:
This line is present in the `master` [branch](https://github.com/sunpy/ndcube/blob/master/ndcube/mixins/sequence_plotting.py#L969), and the git blame shows that you made the changes in this [PR](https://github.com/sunpy/ndcube/blame/d05b8254fa0ffd104c3ff3b763fadaf17fed4ac8/ndcube/mixins/sequence_plotting.py#L969)